action/version.go

```diff
@@ -1,9 +1,31 @@
 package action
 
-import "github.com/urfave/cli"
+import (
+	"fmt"
+	"os"
+
+	"github.com/dominikschulz/github-releases/ghrel"
+	"github.com/fatih/color"
+	"github.com/urfave/cli"
+)
 
 // Version prints the gopass version
 func (s *Action) Version(c *cli.Context) error {
 	cli.VersionPrinter(c)
+
+	if disabled := os.Getenv("CHECKPOINT_DISABLE"); disabled != "" {
+		return nil
+	}
+
+	r, err := ghrel.FetchLatestStableRelease("justwatchcom", "gopass")
+	if err != nil {
+		fmt.Println(color.RedString("\nError checking latest version: %s", err))
+		os.Exit(1)
+	}
+
+	if r.Name != s.version {
+		fmt.Println(color.YellowString("\nYour version of gopass is out of date!\nThe latest version is %s.\nYou can update by downloading from www.justwatch.com/gopass", r.Name))
```

Comment:
I'd like to at least mention package managers to avoid confusion.
```diff
- You can update by downloading from www.justwatch.com/gopass.
+ You can update by downloading from www.justwatch.com/gopass or via your package manager.
```